presto-cli/src/main/java/com/facebook/presto/cli/OutputHandler.java

```diff
@@ -93,5 +96,11 @@ private void flush(boolean complete)
             printer.printRows(unmodifiableList(rowBuffer), complete);
             rowBuffer.clear();
         }
+        printer.flush();
```

Comment:
`CSVPrinter` and `TSVPrinter` doesn't call `flush()` in `printRows()` so let's be safe.